src/sasjs-help/index.js

```diff
@@ -200,6 +200,24 @@ export async function printHelpText() {
           command example: sasjs folder move /Public/sourceFolder /Public/targetFolder
           NOTE: Providing target name (--target targetName or -t targetName) is optional. Default target name will be used if target name was omitted.
 
+    * ${chalk.greenBright('job <command>')} - performs operations on jobs.
+        * ${chalk.cyanBright('execute')} - triggers job for execution.
+          command example: sasjs job execute /Public/job --target targetName
+          command example: sasjs job execute /Public/job -t targetName
+          command example: sasjs job execute /Public/job --target targetName --wait
+          command example: sasjs job execute /Public/job -t targetName -w
+          command example: sasjs job execute /Public/job --target targetName --output
+          command example: sasjs job execute /Public/job -t targetName -o
+          command example: sasjs job execute /Public/job -t targetName -o ./outputFolder
+          command example: sasjs job execute /Public/job --target targetName --wait --output
+          command example: sasjs job execute /Public/job -t targetName -w -o
+          command example: sasjs job execute /Public/job -t targetName --log ./logFolder
+          command example: sasjs job execute /Public/job -t targetName -l ./logFolder
```

Comment:
same issue here, are we specifying a folder or file - we should probably specify a file to allow more control

Reply:
eg ./logFolder/mylog.log

Reply:
fixed